bundle/edu.gemini.spModel.core/src/main/scala/edu/gemini/spModel/core/Magnitude.scala

```diff
@@ -5,20 +5,20 @@ import Scalaz._
 
 case class Magnitude(value: Double, band: MagnitudeBand, error: Option[Double], system: MagnitudeSystem) {
 
-  /** Secondary constructor defaulting to `MagnitudeSystem.VEGA`. */
+  /** Secondary constructor. */
   def this(value: Double, band: MagnitudeBand, error: Double) =
-    this(value, band, Some(error), MagnitudeSystem.VEGA)
+    this(value, band, Some(error), band.defaultSystem)
```

Comment:
These changes are good, this makes them consistent with the old spmodel magnitude changes done a while ago